.travis.yml

```diff
@@ -1,11 +1,20 @@
-language: sh
+language: bash
 
-before_script:
-  - wget "http://ftp.debian.org/debian/pool/main/s/shellcheck/shellcheck_0.3.7-5_amd64.deb"
-  - sudo dpkg -i "shellcheck_0.3.7-5_amd64.deb"
+# Use container-based infrastructure for quicker build start-up
+sudo: false
+
+addons:
+  apt:
+    sources:
+    - debian-sid # Get shellcheck from the Debian repo
```

Comment:
I just grabbed what was on the shellcheck wiki. Not sure what those three suffixes reference or which we should prefer. Would -stable allow us to change this least often?